site/content/blogs/release-v127.en.md

```diff
@@ -87,11 +87,74 @@ Continue with the deployment? (y/N)
 
 If the diff looks good to you, enter "y" to deploy. Otherwise, enter "N" to make adjustments as needed!
 
-## Enable multiple listeners and routing rules for Load Balancers
 
-### Add multiple host-based or path-based routing rules to your Application Load Balancers
+## Enable multiple listeners and listener rules for Load Balancers
+You can now expose multiple ports through your Load Balancers.
+### Add multiple host-based or path-based listener rules to your Application Load Balancer
+To expose multiple ports through the Application Load Balancer, we will configure additional listener rules through a new `http` field called [`additional_rules`](../docs/manifest/lb-web-service.en.md#http-additional-rules). 
```

Comment:
Similar comment here! I feel like we might want to frame all these descriptions as "You can now handle xxx traffic in yyy way" instead of "you can now expose multiple ports". The latter is just a mean to an end, and the end is the former.